BUG: checking for value type when parquet is partitioned (#54074)
pandas-dev/pandas: doc/source/whatsnew/v2.1.0.rst

ExtensionArray
^^^^^^^^^^^^^^

> - Bug in :class:`ArrowStringArray` constructor raises value error on dictionary on values with string type (:issue:`54074`)

Reviewer: ```suggestion
- Bug in :class:`ArrowStringArray` constructor raises ``ValueError`` with dictionary types of strings (:issue:`54074`)
```

Author: Done!
